docs/sources/datasources/jaeger/_index.md

```diff
@@ -43,47 +45,78 @@ Set the data source's basic configuration options carefully:
 | **User**       | Defines the user name for basic authentication.                          |
 | **Password**   | Defines the password for basic authentication.                           |
 
-You can also configure settings specific to the Jaeger data source:
+You can also configure settings specific to the Jaeger data source. These options are described in the sections below.
 
-### Configure trace to logs
+### Trace to logs
 
-{{< figure src="/static/img/docs/explore/traces-to-logs-settings-8-2.png" class="docs-image--no-shadow" caption="Screenshot of the trace to logs settings" >}}
+![Trace to logs settings](/media/docs/tempo/tempo-trace-to-logs-9-4.png)
 
 > **Note:** Available in Grafana v7.4 and higher.
+> If you use Grafana Cloud, open a [support ticket in the Cloud Portal](/profile/org#support) to access this feature.
 
 The **Trace to logs** setting configures the [trace to logs feature]({{< relref "../../explore/trace-integration" >}}) that is available when you integrate Grafana with Jaeger.
 
-**To configure trace to logs:**
+There are two ways to configure the trace to logs feature:
+
+- Use a simplified configuration with default query, or
+- Configure a custom query where you can use a [template language]({{< relref "../../dashboards/variables/variable-syntax">}}) to interpolate variables from the trace or span.
+
+#### Use a simple configuration
+
+1. Select the target data source.
+1. Set start and end time shift. As the logs timestamps may not exactly match the timestamps of the spans in trace it may be necessary to search in larger or shifted time range to find the desired logs.
```

Comment:
```suggestion
1. Set start and end time shift. Since the logs timestamps may not exactly match the timestamps of the spans in trace, it may be necessary to search in larger or shifted time range to find the desired logs.
```